ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -214,3 +215,13 @@ export function isScanConfigurationDisabled(
 
     return false;
 }
+
+export function getTimeDifferenceAsPhrase(iso8601: string, date: Date) {
+    // If initial numeric time is zero before initial scan on creation of scan schedule,
+    // return 0 seconds ago instead of 54 years ago (that is, since beginning of Unix epoch).
+    if (iso8601.slice(0, 10) === '1970-01-01') {
+        return '0 seconds ago';
```

Comment:
Brad, let’s collaborate to reproduce. Depending on duration of edge case, text might need improvement.

Reply:
I was wondering how you'd be able to test this without getting lucky. Simulating it yourself is probably how I would've done it as well but it would be nice to be able to reproduce naturally. Assuming I understood correctly..?